plugin/secondary/README.md

```diff
@@ -38,6 +38,19 @@ When a zone is due to be refreshed (Refresh timer fires) a random jitter of 5 se
 applied, before fetching. In the case of retry this will be 2 seconds. If there are any errors
 during the transfer the transfer fails; this will be logged.
 
+## Metrics
+
+If monitoring is enabled (via the *prometheus* directive) then the following metric is exported:
+
+* `coredns_secondary_last_updated_seconds{server, zone}` - duration of the last update.
+* `coredns_secondary_last_updated_serial{server, zone}` - gauge of serial update attempts.
```

Comment:
I don't understand the gauges here, what are we trying to measure and why aren't we doing some like process_start_time_seconds but then for these updates?

Reply:
even I don't really because I was trying to make sense of these guages from their names that were mentioned in the issue thread by @SuperQ here https://github.com/coredns/coredns/issues/1407#issuecomment-428276568 but not a lot of details were provided about them which made it hard to make sense of what they actually mean.

I could drop the guages for now and just submit this PR with the metrics that you find acceptable and if someone would like to open a discussion about guages again in the future and has a clearer description then I can implement them in another PR. Sounds good?

Reply:
> why aren't we doing some like process_start_time_seconds but then for these updates?

this PR does have `coredns_secondary_last_updated_seconds` which records the duration of the last update. I don't see how `process_start_time_seconds` is different from that cc @miekg

Reply:
last_updated_seconds is the timestamp of when the last update happened, not how long it took.

Reply:
oh, thanks for clearing that up.